BookingsApi/BookingsApi.AcceptanceTests/packages.lock.json

```diff
@@ -157,9 +157,9 @@
       },
       "VH.AcceptanceTests.Common": {
         "type": "Direct",
-        "requested": "[1.21.64, )",
-        "resolved": "1.21.64",
-        "contentHash": "Nr4an9YakJsaYyHTeqpRS05XHvMqmTN+KkMKbWxtuCpUtH1Q5H8Esq41huD2nQBnyJec/hBakWFb8+5thUtOFA==",
+        "requested": "[1.21.68, )",
```

Comment:
Same as above. will this break on the current pipelin?

Reply:
that version does exist in current feed